.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
Curious why we chose to use singleQuote instead of our usual in our other repos. Not that we have a specific style across our current repos but the connect repos default to the default prettier config.

Reply:
https://github.com/TanStack/query/blob/4abce31572171b9b33886f4680adcc780eeec45c/.prettierrc#L3

Reply:
We also discussed this offline. In keeping with the style of tanstack, this makes sense.